### ISSUE TITLE
failure on IPSocket.getaddress

### ISSUE THREAD

**Issue by imtayadeway** [tenderlove/gem_survey#3]
Hi! I'm getting a failure with the following output:

```
-:53:in `getaddress': getaddrinfo: nodename nor servname provided, or not known (SocketError)
    from -:53:in `<main>'
```

in irb:

```
2.2.0 :001 > require 'socket'
 => true
2.2.0 :002 > Socket.gethostname
 => "tmm-mbp-014.domain"
2.2.0 :003 > IPSocket.getaddress(Socket.gethostname)
SocketError: getaddrinfo: nodename nor servname provided, or not known
    from (irb):3:in `getaddress'
    from (irb):3
    from /Users/timwade/.rvm/rubies/ruby-2.2.0/bin/irb:11:in `<main>'
```

i'd love to find a way to submit my results.
thanks!

**Comment by tenderlove** [comment]
Should be fixed in 8afe81478c5da9dddfc2334ecd6517d1d1a8dfa8

**Comment by imtayadeway** [comment]
that worked like a charm :smile: 

thanks!
